man/systemd-stub.xml

```diff
@@ -167,6 +167,15 @@
       details on encrypted credentials. The generated <command>cpio</command> archive is measured into TPM
       PCR 12 (if a TPM is present).</para></listitem>
 
+      <listitem><para>Similarly, files
```

Comment:
oh, ouch.

can you add a "Follow-up: ac32323271305252935469723e460139a65ed378" to the commit msg, because I broke the logic since then